src/antsibull_docs/test_helpers.py

```diff
@@ -0,0 +1,39 @@
+# Author: Felix Fontein <felix@fontein.de>
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or
+# https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+# SPDX-FileCopyrightText: 2023, Ansible Project
+"""Detect and setup for CI tests"""
+
+from __future__ import annotations
+
+import os
+
+import antsibull_docs
+
+# Use an environment variable with a special value as a CI marker
+ANTSIBULL_DOCS_CI_ENV_MARKER_NAME = "_ANTSIBULL_DOCS_CI_MARKER"
+ANTSIBULL_DOCS_CI_ENV_MARKER_VALUE = "Noo4oogongae"
+
+# The version string to use when in CI
+ANTSIBULL_DOCS_CI_VERSION = "<ANTSIBULL_DOCS_VERSION>"
+
+
+def is_in_antsibull_ci():
+    """
+    Determine whether the code is running in CI.
+    """
+    return (
+        os.environ.get(ANTSIBULL_DOCS_CI_ENV_MARKER_NAME)
+        == ANTSIBULL_DOCS_CI_ENV_MARKER_VALUE
+    )
+
+
+def ci_setup():
```

Comment:
@gotmax23 I'm not really happy about this mechanism. I wanted to avoid some manual patching of `../../src/antsibull_docs/__init__.py` in `tests/functional/build-*.sh`. But maybe it's better to just do exactly that? What do you think?

Reply:
You could create a shim Python script that imports `antsibull_docs`, monkeypatches `antsibull_docs.__version__`, and then calls `sys.exit(antsibull_docs.cli.antsibull_docs.main())`

Reply:
Right. I wonder why I didn't think of that yesterday...

Reply:
Ah wait, it wasn't yesterday, but today, early morning. Maybe that explains it ;)

Reply:
Implemented in 39adce6eff12b8ebef8343c94aa332c2786d8681.